.env.sample

```diff
@@ -0,0 +1,6 @@
+export SOLIDUS_BRANCH=v2.10
+export DB=postgresql
+export RAILS_VERSION="~> 5.2.0"
```

Comment:
There are already defaults in the Gemfile. If you want to update them you should do that there.